source/fem-dem/parameters_cfd_dem.cc

```diff
@@ -40,6 +40,11 @@ namespace Parameters
                       "1",
                       Patterns::Double(),
                       "The upper bound for void fraction L2 projection");
+    prm.declare_entry(
+      "number of particle refinement",
+      "0",
+      Patterns::Double(),
+      "The refinement order of the particle for the satellite point method");
```

Comment:
If you decide to change the name, here it could be something like "The refinement factor used to calculate the number of pseudo-particles in the satellite point method"